src/main/java/rails/game/action/ExchangeTokens.java

```diff
@@ -82,11 +82,4 @@ public String toString() {
         ;
     }
 
-    /** Deserialize */
-    private void readObject(ObjectInputStream in) throws IOException,
-            ClassNotFoundException {
-
-        in.defaultReadObject();
-
-    }
 }
```

Comment:
Hm no replacement isnt hat Object needed ?

Reply:
`readObject()` is only needed if you are going to customize the reading-- in this case we are just turning around and calling the default reader.